modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/interceptor/CmmnCommandInvoker.java

```diff
@@ -76,9 +76,7 @@ protected void executeOperation(CommandContext commandContext, boolean isStoreCa
         if (runnable instanceof CmmnOperation) {
             CmmnOperation operation = (CmmnOperation) runnable;
 
-            if (logger.isDebugEnabled()) {
```

Comment:
Let's leave the if check here. When doing a call to a method with varArgs an array is instantiate on every call. This is really bad for performance and memory efficiency. Doing an if check on the other side is much better. 

This places is a hot path that would be affected by the unnecessary varArgs array creation.

Reply:
Done